horace_core/sqw/@sqw/private/cut_accumulate_data_.m

```diff
@@ -225,6 +221,7 @@
         [npix, s, e, pix_ok, unique_runid_l, pix_indx, selected] = ...
             targ_proj(i).bin_pixels(targ_axes(i), candidate_pix, npix, s, e);
 
+        candidate_pix = sym.transform_pix(candidate_pix, {}, selected);
         candidate_pix = candidate_pix.tag(selected);
```

Comment:
As usual, do not understand the logic here. You loop over projections which project various areas of the sqw file to target area. But use only one sym operation for transforming pixels, while each projection should have its own. 
werid.

Reply:
Fixed that